plugins/modules/packaging/os/pkgng.py

```diff
@@ -23,6 +23,13 @@
     name:
         description:
             - Name or list of names of packages to install/remove.
+            - "With I(name=*), I(state: latest) will operate, but I(state: present) and I(state: absent) will be noops."
+            - >
+                Warning: In ansible 2.9 and earlier this module had a misfeature
```

Comment:
```suggestion
                Warning: In Ansible 2.9 and earlier this module had a misfeature
```
Sorry for not noticing that earlier!